packages/twenty-front/src/modules/ui/layout/selectable-list/components/SelectableItem.tsx

```diff
@@ -1,8 +1,14 @@
 import { ReactNode, useEffect, useRef } from 'react';
+import styled from '@emotion/styled';
 import { useRecoilValue } from 'recoil';
 
 import { useSelectableList } from '@/ui/layout/selectable-list/hooks/useSelectableList';
 
+const StyledSelectableItem = styled.div`
+  height: 100%;
```

Comment:
I don't think this should be the responsibility of SelectableItem.
You can style SelectableItem from the outside instead (as we pass className as prop)